contracts/ethereum-sc/contracts/ERC20Safe.sol

```diff
@@ -24,7 +24,7 @@ contract ERC20Safe {
     uint256 public depositsCount;
     uint256 public batchesCount;
     uint256 public batchTimeLimit = 10 minutes;
-    uint256 public batchSettleLimit = 10 minutes;
+    uint256 public batchSettleBlockCount = 24;
```

Comment:
10 minutes equal with 40 ?

Reply:
Ah, nope. I'll update this to 600 seconds / 15 seconds